src/js/utils/string.js

```diff
@@ -0,0 +1,2 @@
+export const toTitleCase = (str) =>
+  str.replace(/\w\S*/g, (txt) => txt.charAt(0).toUpperCase() + txt.substring(1).toLowerCase());
```

Comment:
Not sure it is used again, but if you ever need to do this, consider the CSS alternative `text-transform: capitalize`

Reply:
It is not used, therefore removed it. It was previously used for the older version of provenance but my bad missed it in the cleanup.